cg/store/crud/update.py

```diff
@@ -16,3 +16,11 @@ def __init__(self, session: Session):
     def update_flow_cell_has_backup(self, flow_cell: Flowcell, has_backup: bool) -> None:
         flow_cell.has_backup = has_backup
         self.session.commit()
+
+    def add_sample_comment(self, sample: Sample, comment: str) -> None:
```

Comment:
Probably not for this PR, but would this not just be an "update" as the comment is already there but with value "none"? 

```suggestion
    def update_sample_comment(self, sample: Sample, comment: str) -> None:
```

Reply:
Fixed